### ISSUE TITLE
Order of jQuery/popper file includes backwards

### ISSUE THREAD

**Issue by catchmyfame** [twbs/bootstrap#30873]
On the page https://getbootstrap.com/docs/4.5/getting-started/download/#bootstrapcdn it says:

> If you’re using our compiled JavaScript, don’t forget to include CDN versions of jQuery and Popper.js before it.

It then shows an example of

```
<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
```

See the issue? The example shows popper _after_ jQuery, but the docs say it should come _before_ it.

The error appears to exist in the description, and not the example. I suggest changing:

> If you’re using our compiled JavaScript, don’t forget to include CDN versions of jQuery and Popper.js before it.

to:

> If you’re using our compiled JavaScript, don’t forget to include CDN versions of jQuery and Popper.js after it.

**Comment by ffoodd** [comment]
The wording might be confusing but the examples are right: "it" at the end of the sentence refers to "our compiled JavaScript", not jQuery.

Not sure about the best way to make this less confusing, though.

**Comment by catchmyfame** [comment]
> The wording might be confusing but the examples are right: "it" at the end of the sentence refers to "our compiled JavaScript", not jQuery.
> 
> Not sure about the best way to make this less confusing, though.

Right, that's my point, the grammar/wording is misleading and should be changed. I'm not an English major but it seems like a dangling participle. If the above suggestion doesn't seem more clear, then how about:

> If you’re using our compiled JavaScript, don’t forget to include the CDN versions of jQuery followed by Popper.js.

Or perhaps:

> If you’re using our compiled JavaScript, don’t forget to include the CDN versions of jQuery and Popper.js, in that order.

**Comment by ffoodd** [comment]
So I've been even more confusing I guess! 

The order of jQuery and Popper.js doesn't matter: the sentence is meant to tell that both should come before our compiled JavaScript.

So both of those will work fine:
1. jQuery, Popper.js, Bootstrap
2. Popper.js, jQuery, Bootstrap

**Comment by mc2002tii** [comment]
It seems pretty clear to me that 'it' refers to 'our compiled JavaScript' and not 'jQuery,' especially with the example there.  I suppose you could change 'before it' to 'first' to remove any possible ambiguity.

**Comment by XhmikosR** [comment]
The order only matters in what @ffoodd said https://github.com/twbs/bootstrap/issues/30873#issuecomment-631561947

The rest are common script inclusion principles. The order matters if the next script depends on another one, but that's not the case with jQuery and popper. *We* depend on jQuery and popper.

This reminds me, we should redo all the getting started snippets to mention our bundle. That's what most people should use.

**Comment by XhmikosR** [comment]
Fixed in #30896
